File: PB - More Exercises Python/5.0 For-Loop - More Exercises/06. Bills.py
months_count = int(input())
all_electricity = sum([float(input()) for _ in range(months_count)])
water = 20 * months_count
internet = 15 * months_count
other = all_electricity * 1.20  + water * 1.20  + internet * 1.2
average = (all_electricity + water + internet + other) / months_count

print(f'Electricity: {all_electricity:.2f} lv')
print(f'Water: {water:.2f} lv')
print(f'Internet: {internet:.2f} lv')
print(f'Other: {other:.2f} lv')
print(f'Average: {average:.2f} lv')


'''
5
68.63
89.25
132.53
93.53
63.22
'''

File: PB - More Exercises GO/5.0 For-Loop - More Exercises/06. Bills.go
package main

import "fmt"

func main() {
	var monthsCount, allElectricity float64
	fmt.Scan(&monthsCount)
	for i := 0; i < int(monthsCount); i++ {
		var current float64
		fmt.Scan(&current)
		allElectricity += current
	}
	water := 20 * monthsCount
	internet := 15 * monthsCount
	other := allElectricity*1.20 + water*1.20 + internet*1.2
	average := (allElectricity + water + internet + other) / monthsCount

	fmt.Printf("Electricity: %.2f lv\n", allElectricity)
	fmt.Printf("Water: %.2f lv\n", water)
	fmt.Printf("Internet: %.2f lv\n", internet)
	fmt.Printf("Other: %.2f lv\n", other)
	fmt.Printf("Average: %.2f lv\n", average)
}

/*
name   :06. Bills
input  :5 68.63 89.25 132.53 93.53 63.22
output :
Electricity: 447.16 lv
Water: 100.00 lv
Internet: 75.00 lv
Other: 746.59 lv
Average: 273.75 lv
*/
